src/Transactions/certificates.ts

```diff
@@ -0,0 +1,208 @@
+     , 'CertRegKey' as "formalType"
+     , reg.tx_id as "txId"
+     , reg.cert_index as "certIndex"
+     , encode(addr.hash,'hex') as "stakeCred"
+     , null::text as "poolHashKey"
+     , null::text as "poolParamsOperator"
+     , null::text as "poolParamsVrfKeyHash"
+     , null::numeric as "poolParamsPledge"
+     , null::bigint as "poolParamsCost"
+     , null::double precision as "poolParamsMargin"
+     , null::text as "poolParamsRewardAccount"
+     , null::json as "poolParamsOwners"
+     , null::json as "poolParamsRelays"
+     , null as "poolParamsMetaDataUrl"
+     , null::text as "poolParamsMetaDataHash"
+     , null::integer as "epoch"
+     , null as "mirPot"
+     , null::json as "rewards"
+from stake_registration as reg
+join stake_address as addr 
+  on reg.addr_id = addr.id
+
+UNION ALL
+  
+select 'StakeDeregistration' as "jsType"
+     , 'CertDeregKey' as "formalType"
+     , dereg.tx_id as "txId"
+     , dereg.cert_index as "certIndex"
+     , encode(addr.hash,'hex') as "stakeCred"
+     , null::text as "poolHashKey"
+     , null::text as "poolParamsOperator"
+     , null::text as "poolParamsVrfKeyHash"
+     , null::numeric as "poolParamsPledge"
+     , null::bigint as "poolParamsCost"
+     , null::double precision as "poolParamsMargin"
+     , null::text as "poolParamsRewardAccount"
+     , null::json as "poolParamsOwners"
+     , null::json as "poolParamsRelays"
+     , null as "poolParamsMetaDataUrl"
+     , null::text as "poolParamsMetaDataHash"
+     , null::integer as "epoch"
+     , null as "mirPot"
+     , null::json as "rewards"
+from stake_deregistration as dereg
+join stake_address as addr
+  on dereg.addr_id = addr.id
+
+UNION ALL
+  
+select 'StakeDelegation' as "jsType"
+     , 'CertDelegate' as "formalType"
+     , del.tx_id as "txId"
+     , del.cert_index as "certIndex"
+     , encode(addr.hash,'hex') as "stakeCred"
+     , encode(pool_hash.hash, 'hex') as "poolHashKey"
+     , null::text as "poolParamsOperator"
+     , null::text as "poolParamsVrfKeyHash"
+     , null::numeric as "poolParamsPledge"
+     , null::bigint as "poolParamsCost"
+     , null::double precision as "poolParamsMargin"
+     , null::text as "poolParamsRewardAccount"
+     , null::json as "poolParamsOwners"
+     , null::json as "poolParamsRelays"
+     , null as "poolParamsMetaDataUrl"
+     , null::text as "poolParamsMetaDataHash"
+     , null::integer as "epoch"
+     , null as "mirPot"
+     , null::json as "rewards"
+from delegation as del
+join stake_address as addr
+  on del.addr_id = addr.id
+join pool_update 
+  on del.update_id = pool_update.id
+join pool_hash
+  on pool_update.hash_id = pool_hash.id
+
+UNION ALL
+  
+select 'PoolRegistration' as "jsType"
+     , 'CertRegPool' as "formalType"
+     , pool.registered_tx_id as "txId" 
+     , pool.cert_index as "certIndex"
+     , null as "stakeCred"
+     , null::text as "poolHashKey"
+     , encode(pool_hash.hash,'hex') as "poolParamsOperator" 
```

Comment:
I'm not 100% sure this is the correct value for pool operator, as mentioned in the PR message.

Reply:
Probably the best thing to do is just to query one of the entries in our DB and the look it up on https://adapools.org to see if the value matches